### PR TITLE
MGDOBR-476 Implement test for periodically checking of long running…

### PR DESCRIPTION
**Pull request by sutaakar**
… bridge instance

Signed-off-by: Karel Suta <ksuta@redhat.com>

[MGDOBR-476](https://issues.redhat.com/browse/MGDOBR-476)

Adjusted IngressSteps implementation to be readable and more flexible.
Added dedicated test suite for periodical Slack test to keep it separate as this test is meaningful only for dev cluster.
Store cloud event id in test context instead of Bridge context to allow using cloud event references without Bridge.

Next PR will create a GHA script to run this test against dev cluster.

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -83,6 +83,41 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <systemProperties>
+            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```
Comment (jiripetrlik) [inline integration-tests/pom.xml:91]: Is this related to periodic check?

File: integration-tests/pom.xml [hunk 2]
```diff
@@ -83,6 +83,41 @@
...
+            <!--cucumber.filter.tags>${tags}</cucumber.filter.tags-->
+
+            <!-- https://github.com/cucumber/cucumber-jvm/tree/main/junit-platform-engine#parallel-execution -->
+            <cucumber.execution.parallel.enabled>${parallel}</cucumber.execution.parallel.enabled>
+            <cucumber.junit-platform.naming-strategy>long</cucumber.junit-platform.naming-strategy>
+            <cucumber.execution.parallel.config.fixed.parallelism>${parallel.concurrency.number}</cucumber.execution.parallel.config.fixed.parallelism>
+
+            <!-- Token authorization configuration-->
+            <bridge.client.id>kermit</bridge.client.id>
+            <bridge.client.secret>N5TW1EfuIcQsplRsLXJ1aE3DZZMPN5ZH</bridge.client.secret>
```
Comment (jiripetrlik) [inline integration-tests/pom.xml:105]: Do we want to hard code this parameter?

Comment (sutaakar) [reply integration-tests/pom.xml:105]: This value is taken from https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json to simplify usage and it is relatively complicate to share it just on one place.
So, I would say yes, in this case it is ok to hardcode it here.

**Review by Srihari1192 — APPROVED** [review]
LGTM

File: integration-tests/pom.xml (hunk 2 above)
Comment (jomarko) [reply integration-tests/pom.xml:104]: @sutaakar  shouldn't we share this information as comment in pom.xml? I mean where `N5TW1EfuIcQsplRsLXJ1aE3DZZMPN5ZH` is coming from?

Comment (sutaakar) [reply integration-tests/pom.xml:104]: Yes, good idea.

Comment (sutaakar) [reply integration-tests/pom.xml:104]: Created https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/674
